src/components/GroupList/GroupItem.js

```diff
@@ -0,0 +1,68 @@
+/* @flow */
+
+import React, { PropTypes, Component } from 'react'
+import Divider from 'material-ui/Divider'
+import Avatar from 'material-ui/Avatar'
+import FileFolder from 'material-ui/svg-icons/file/folder'
+import { ListItem } from 'material-ui/List'
+
+class GroupItem extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      open: false,
+      followed: false,
+    }
+    this.toggleOpen = this.toggleOpen.bind(this)
+    this.toggleFollow = this.toggleFollow.bind(this)
+  }
+
+  toggleOpen() {
+    const value = this.state.open
+    this.setState({ open: !value })
+  }
+
+  toggleFollow() {
+    const value = this.state.followed
+    this.setState({ followed: !value })
+  }
+
+  render() {
+    const {
+      item,
+      primaryTextProp,
+      secondaryTextProp,
+      valueProp,
+      clickHandler,
+    } = this.props
+
+    const primaryText = item[primaryTextProp] || 'test'
+    const secondaryText = secondaryTextProp ? item[secondaryTextProp] : ''
+    const value = item[valueProp]
+
+    return (
+      <div>
+        <ListItem
+          value={value}
+          onTouchTap={() => clickHandler(value)}
```

Comment:
Avoid creating functions for events handling inside render function, it leads to performance issue, because every time the component rerendered new function is created, which is expensive operation, the following implementation is preferable : `onTouchTap={clickHandler}` or even better 
```
function touchTapEventHandler(value) {
   if (clickHandler) {
      clickHandler(value)
   }
 }

...

onTouchTap={this.touchTapEventHandler}

```
There are lots of stubs like this, we should keep replacing them with proper implementation.